rust/processor/src/processors/objects_processor.rs

```diff
@@ -119,12 +119,6 @@ fn insert_current_objects_query(
             .on_conflict(object_address)
             .do_update()
             .set((
-                owner_address.eq(excluded(owner_address)),
-                state_key_hash.eq(excluded(state_key_hash)),
-                allow_ungated_transfer.eq(excluded(allow_ungated_transfer)),
-                last_guid_creation_num.eq(excluded(last_guid_creation_num)),
-                last_transaction_version.eq(excluded(last_transaction_version)),
-                is_deleted.eq(excluded(is_deleted)),
```

Comment:
you can't remove this. I meant the objects table

Reply:
the objects table already has it set